docs/source/tracking.rst

```diff
@@ -230,13 +228,47 @@ The UI contains the following key features:
 Querying Runs Programmatically
 ==============================
 
-All of the functions in the Tracking UI can be accessed programmatically. This makes it easy to do several common tasks:
+You can access all of the functions in the Tracking UI programmatically. This makes it easy to do several common tasks:
 
 * Query and compare runs using any data analysis tool of your choice, for example, **pandas**. 
 * Determine the artifact URI for a run to feed some of its artifacts into a new run when executing a workflow. For an example of querying runs and constructing a multistep workflow, see the MLflow `Multistep Workflow Example project <https://github.com/mlflow/mlflow/blob/15cc05ce2217b7c7af4133977b07542934a9a19f/examples/multistep_workflow/main.py#L63>`_.
 * Load artifacts from past runs as :ref:`models`. For an example of training, exporting, and loading a model, and predicting using the model, see the MLFlow `TensorFlow example <https://github.com/mlflow/mlflow/tree/master/examples/tensorflow>`_.
 * Run automated parameter search algorithms, where you query the metrics from various runs to submit new ones. For an example of running automated parameter search algorithms, see the MLflow `Hyperparameter Tuning Example project <https://github.com/mlflow/mlflow/blob/master/examples/hyperparam/README.rst>`_.
 
+.. _artifact-locations:
+
+Referencing Artifacts
+---------------------
+
+When you specify the location of an artifact in MLflow APIs, the syntax depends on whether you
+are invoking the Tracking API or Models API. For the Tracking API, you specify the artifact location using a (run ID, relative path) tuple. 
```

Comment:
The `Projects API` also uses the same syntax as the `Models API`; we should clarify that here.